sample-code/src/server.ts

```diff
@@ -265,3 +273,95 @@ app.get('/langchain/invoke-rag-chain', async (req, res) => {
       .send('Yikes, vibes are off apparently 😬 -> ' + error.message);
   }
 });
+
+/* Document Grounding */
+app.get(
+  '/document-grounding/invoke-orchestration-grounding',
+  async (req, res) => {
+    try {
+      res.setHeader('Content-Type', 'text/event-stream');
+      res.setHeader('Connection', 'keep-alive');
+      res.flushHeaders();
```

Comment:
[q] Aren't these headers specifically for streaming responses? Isn't  `Content-Type =text/plain` enough here?

Or is it just because you wanted to send the response for each call as and when they are done?

Reply:
I want the browser to immediately print out some string on the fly so that user doesn't have to wait for all requests to be finished and see the result, which could take many seconds.